nixos/modules/services/web-servers/phpfpm/pool-options.nix

```diff
@@ -0,0 +1,57 @@
+{ lib, config }:
+
+let
+  fpmCfg = config.services.phpfpm;
+in
+
+with lib; {
+
+  options = {
+
+    listen = mkOption {
```

Comment:
```suggestion
    listen = mkOption {
      type = types.str;
      example = "/path/to/unix/socket";
      default = "/run/phpfpm/${poolName}.sock"; # TODO Get the pool name from somewhere
      defaultText = "/run/phpfpm/$poolName.sock";
      description = ''…'';
    };
```

Reply:
This change would allow us to drop many of the `listen` configs and somehow streamline the socket locations for future modules

Reply:
@dasJ on second thought I'm going to save this for a follow up PR i have in mind.